src/app/routines/exercisesTable/ExercisesTable.tsx

```diff
@@ -0,0 +1,38 @@
+import { TableContainer, Table, TableHead, TableRow, TableCell, TableBody } from '@mui/material';
+import ExercisesTableRow from './ExercisesTableRow';
+import { useRoutineExercises } from '@/app/common/context/RoutineExercisesContext';
+
+interface ExerciseTableProps {
+  editable: boolean;
+}
+
+const ExercisesTable = ({ editable = true }: ExerciseTableProps) => {
+  const { exercises } = useRoutineExercises();
+
+  return (
+    <>
+      <TableContainer>
+        <Table aria-label='collapsible table'>
```

Comment:
minor, it would be better to have aria-label that describes that its exercise table.